docs/hassio_debugging.md

```diff
@@ -31,10 +31,10 @@ ssh root@hassio.local -p 22222
 
 ```bash
 # Logs from the supervisor service on the Host OS
-journalctl -f -u resin-supervisor.service
+journalctl -f -u hassos-supervisor.service
```

Comment:
Will this work for Hass.io users as well.

Reply:
No the resinOS based Hassio will still need that one. I wasn't sure whether to leave them and note "deprecated" or to replace them at the time.